Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM gentoo/stage3-amd64:latest
+FROM gentoo/stage3:amd64
 MAINTAINER Alex Brandt <alunduil@gentoo.org>
 
 # Set a locale
-RUN sed -e '/en_US/s/^#//' -i /etc/locale.gen
+RUN echo en_US.UTF8 UTF-8 >> /etc/locale.gen
+RUN echo en_US ISO-8859-1 >> /etc/locale.gen
```

Comment:
Why echo a locale and then set the locale?

Reply:
Default for locale.gen is C. So, to set en_US-utf8, we first need to compile it by echoing it to locale.gen and running locale-gen, and only then we can set it. I hope that made sense.

Reply:
> Default for locale.gen is C. So, to set en_US-utf8, we first need to compile it by echoing it to locale.gen and running locale-gen, and only then we can set it. I hope that made sense.

Yep, that sounds like something to add as a comment on those lines though.  Otherwise it's easily forgotten next time we peak at it.